subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt

```diff
@@ -254,9 +262,9 @@ class InstrumentationTestsPlugin : Plugin<Project> {
                                 }
 
                                 val isFullTestSuite = gitState.map {
-                                        it.isOnDefaultBranch
-                                                && instrumentationConfiguration.impactAnalysisPolicy is ImpactAnalysisPolicy.Off
-                                    }
+                                    it.isOnDefaultBranch
+                                        && instrumentationConfiguration.impactAnalysisPolicy is ImpactAnalysisPolicy.Off
```

Comment:
hm, where is the impact analysis module in the general scheme of Avito runner?

Reply:
it is a part of the tests filtration; should be optional here